File: old/cAPS lOCK/1.java
import java.util.Scanner;
import java.lang.Character;

public class cAPSlOCK{
	public static int chk(String word){
		int i,n=0;
		char[] wordarray = word.toCharArray();
		for(i=0;i<word.length();i++){
			if(Character.isLowerCase(wordarray[i])){
				n++;
			}
		}
		return n;
	}
	public static void main(String[] arg){
		Scanner in = new Scanner(System.in);
		String word;
		int i;
		word = in.nextLine();
		char[] wordarray = word.toCharArray();
		if(Character.isLowerCase(wordarray[0]) && chk(word)>1){
			System.out.print(word);
		}
		else if(Character.isLowerCase(wordarray[0]) && chk(word)<=1){
			wordarray[0] = Character.toUpperCase(wordarray[0]);
			System.out.print(wordarray[0]);
			for(i=1;i<word.length();i++){
				wordarray[i] = Character.toLowerCase(wordarray[i]);
				System.out.print(wordarray[i]);
			}
		}
		else if(word==word.toUpperCase()){
			System.out.print(word.toLowerCase());
		}
		else{
			System.out.print(word);
		}

	}
}

File: old/cAPS lOCK/1.py
def chk(word):
	n=0
	for i in word:
		if i==i.lower():
			n+=1
	return n

x = raw_input()
if x[0]==x[0].lower() and chk(x)>1:
	print x
elif x[0]==x[0].lower() and chk(x)<=1:
	print x[0].upper()+x[1:].lower()
elif x==x.upper():
	print x.lower()
else:
	print x

